src/Components/ClustersListTable/ClustersListTable.spec.ct.js

```diff
@@ -541,12 +542,16 @@ describe('clusters list table', () => {
   });
 
   it('names of rows are links', () => {
-    cy.getFirstRow()
-      .find('td[data-label=Name]')
-      .find(
-        `a[href="/clusters/${namedClustersDefaultSorting[0]['cluster_id']}"]`
-      )
-      .should('have.text', namedClustersDefaultSorting[0]['name']);
+    cy.get(TBODY)
+      .children()
+      .each(($el, index) => {
+        cy.wrap($el)
+          .find('td[data-label=Name]')
+          .find(
+            `a[href="/clusters/${namedClustersDefaultSorting[index]['cluster_id']}"]`
```

Comment:
does this selector really yield anything? the things is, here, you use `href=` which is an "exact equality" operator. Whereas, I would expect, you would use "*=" here as you did in https://github.com/RedHatInsights/ocp-advisor-frontend/pull/241/files#diff-11510ebd9fcd477bd2c0ab72124a29e1b5a7a8ec9743a472cbc67852ba71b74aR145

Reply:
Yes, it seems to be working. In this case the match is exact as it is just a redirection to the cluster, without any other default set up.

![image](https://user-images.githubusercontent.com/24312587/166223763-893b5330-978a-4ae8-bd6c-5bf75111d13d.png)

Reply:
Ah, okay, just checked out that `href` can also equal to a pathname, not the full URL 👍🏼